chia/cmds/data.py

```diff
@@ -281,10 +300,10 @@ def add_missing_files(ids: Optional[str], override: bool, foldername: Optional[s
     )
 
 
-@data_cmd.command("add_mirror", short_help="")
-@click.option("-i", "--id", help="", type=str, required=True)
-@click.option("-a", "--amount", help="", type=int, required=True)
-@click.option("-u", "--url", "urls", help="", type=str, multiple=True)
+@data_cmd.command("add_mirror", short_help="Publish mirror urls on chain")
+@click.option("-i", "--id", help="Store id", type=str, required=True)
+@click.option("-a", "--amount", help="Amount for this mirror", type=int, required=True)
+@click.option("-u", "--url", "urls", help="List of urls published for the coin", type=str, multiple=True)
```

Comment:
Especially given that we have cases where you do actually provide a JSON array (list), I wouldn't use list here.  Maybe this...?

```suggestion
@click.option("-u", "--url", "urls", help="URL to publish on the new coin, multiple accepted and will be published to a single coin.", type=str, multiple=True)
```